arch/x86/kvm/vmx/pkvm/hyp/pgtable.c

```diff
@@ -66,6 +66,19 @@ static bool leaf_mapping_allowed(struct pkvm_pgtable_ops *pgt_ops,
 	return leaf_mapping_valid(pgt_ops, vaddr, vaddr_end, pgsz_mask, level);
 }
 
+static void *pgtable_alloc_page(struct pkvm_mm_ops *mm_ops)
+{
+	void *page;
```

Comment:
Uninitialized value of `page` may be used at line 76, as well as returned at line 79.
BTW I found this issue thanks to `-Werror,-Wsometimes-uninitialized` enabled in CrOS SDK build.

Reply:
Thanks. Will fix this.

Reply:
Fixed in the new version of pull request.